src/Patches/PlayerCharacterPatches.cs

```diff
@@ -202,13 +202,25 @@ public class PlayerCharacterPatches {
                     Inventory.GetItemByName("Sword").Quantity = 1;
                     SaveFile.SetInt("randomizer started with sword", 1);
                 }
-                if (TunicRandomizer.Settings.EntranceRandoEnabled)
-                {
+                switch (TunicRandomizer.Settings.FixedLaurelsOption) {
```

Comment:
This can be simplified to just
```suggestion
                SaveFile.SetInt("randomizer laurels location", (int)TunicRandomizer.Settings.FixedLaurelsOption);
```